newswires/client/src/Item.tsx

```diff
@@ -29,6 +29,23 @@ export const Item = ({
 	const [isShowingJson, setIsShowingJson] = useState<boolean>(false);
 	const isSmallScreen = useIsWithinBreakpoints(['xs', 's']);
 
+	const headingRef = useRef<HTMLHeadingElement>(null);
+
+	// scroll to heading when a new item is selected
+	useEffect(() => {
+		if (headingRef.current) {
+			headingRef.current.scrollIntoView({
+				behavior: 'smooth',
```

Comment:
We can probably skip the smooth transition because users will already expect the story to be at the top by default.

Reply:
sounds good, I've updated the commit removing this prop 👍